server/src/main/java/no/nav/foreldrepenger/vtp/server/MockServer.java

```diff
@@ -293,8 +292,7 @@ private boolean tjenesteDisabled(VTPTjeneste tjeneste){
         SOAP("VTP_SOAP"),
         REST("VTP_REST"),
         KAFKA("VTP_KAFKA"),
-        LDAP("VTP_LDAP"),
-        OAUTH2("VTP_OAUTH2");
+        LDAP("VTP_LDAP");
```

Comment:
Hva om du kalte enum-elementene VTP_SOAP etc og ikke bare SOAP, kunne du da spare deg for dette ekstra argumentet ?